documentation/binding-data/tutorial-flow-components-binder-beans.asciidoc

```diff
@@ -86,37 +98,59 @@ binder.forField(yearOfBirthField)
   .bind(Person::getYearOfBirth, Person::setYearOfBirth);
 ----
 
-Note that to use Bean Validation annotations you should have a JSR 303 implementation like Hibernate Validator available in your classpath. E.g. if you use Maven, the following dependency can be used.
+==== Defining Constraints on the Bean Level
 
-[source, xml]
+// The original text was difficult to understand. I copied this code example from the Hibernate docs and referenced them as the source. The original text just linked to the example. We could leave it like this or amend the code. I'm very unsure about the explanation here.   
+
+As an alternative to defining constraint annotations for specific properties, you can define constraints on the bean level.
+
+*Example*: Defining bean-level constraint annotations. _Source: https://docs.jboss.org/hibernate/validator/5.4/reference/en-US/html_single/#validator-gettingstarted-createmodel[Hibernate Validator 5.4.2 - Applying constraints]_
+
+[source, java]
 ----
-<dependency>
-    <groupId>org.hibernate</groupId>
-    <artifactId>hibernate-validator</artifactId>
-    <version>5.4.1.Final</version>
-</dependency>
+package org.hibernate.validator.referenceguide.chapter01;
```

Comment:
I have no idea why build fails on this code snippet, but let's see if we need this at all...